spec/rspec/its_spec.rb

```diff
@@ -376,5 +378,39 @@ def terminator
         end
       end
     end
+
+    describe "#its" do
+      it_behaves_like 'its'
+    end
+
+    describe "#fits" do
+      it_behaves_like 'its'
```

Comment:
This is kind of useless, since the shared example is using `its` all around.

Reply:
Would you prefer it to be removed from both `#fits` and `#xits` blocks and shared_examples reverted?

Reply:
I don't mind it either way. `fits`/`xits` needs to be tested, and this shared example doesn't cover the part that they have in common with `its` - it tests `its`, not `fits`/`xits`.

Reply:
Any tests for this feature should test that focus or skip is applied to an example, and not just stun out the interface, thus there is no need for shared examples and this change should be reverted.

Reply:
Removed the shared_examples and added some additional testing to both xits and fits blocks